server/app/registry.ts

```diff
@@ -32,11 +36,20 @@ logger.add(
   })
 );
 
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: config.get<string>('email'),
+    pass: process.env.EMAIL_PASS,
```

Comment:
add this in the default.js file and use config.get<string>('pass')

Reply:
do you mean for line 43?

Reply:
ya dont call process.env directly, use config like we do everywhere else